lib/modules/manager/gomod/artifacts.ts

```diff
@@ -113,14 +116,23 @@ function addAuthFromHostRule(
 
 function getUpdateImportPathCmds(
   updatedDeps: PackageDependency[],
-  { constraints, newMajor }: UpdateArtifactsConfig
+  { constraints }: UpdateArtifactsConfig
 ): string[] {
   const updateImportCommands = updatedDeps
-    .map((dep) => dep.depName!)
-    .filter((x) => !x.startsWith('gopkg.in'))
+    .map(({ depName, newVersion }) => ({
+      depName: depName!,
+      newMajor: major(newVersion!),
```

Comment:
Is `newMajor` not part of this object already? It might be pre-calculated, simplifying this step

Reply:
`newMajor` is not part of this object as far as I can tell.